src/v/net/server.cc

```diff
@@ -151,10 +151,17 @@ void server::print_exceptional_future(
               ctx,
               address,
               ex);
+        } else if (is_message_error(ex)) {
+            vlog(
+              _log.warn,
+              "Message Error[{}] remote address: {} - {}",
+              ctx,
+              address,
+              ex);
         } else {
-            // Authentication exceptions are logged at WARN, not ERROR, because
-            // they generally point to a misbehaving client rather than a fault
-            // in the server.
+            // Authentication and Message exceptions are logged at WARN, not
+            // ERROR, because they generally point to a misbehaving client
+            // rather than a fault in the server.
```

Comment:
Also a pre-existing issue:

Before the changes, this message would make more sense in the authentication error path, not the "else" path...
Now that it reflects both the authentication and the new path, i would say that it fits better outside the authentication path. Probs. line 147

Reply:
no longer touching this, so leaving it as is